recipe/meta.yaml

```
@@ -29,6 +29,8 @@ requirements:
   run:
     - python
     - ruamel.yaml.clib >=0.2.7  # [py<313]
```

```suggestion
    - ruamel.yaml.clib >=0.2.7
```
I'm not seeing why we can't build for 313 since it exists in linked PR. Does ruamel.yaml not have 313 config?

---

I don't know why but it's an upstream requirement https://inspector.pypi.io/project/ruamel-yaml/0.18.10/packages/ea/46/f44d8be06b85bc7c4d8c95d658be2b68f27711f279bf9dd0612a5e4794f5/ruamel.yaml-0.18.10.tar.gz/ruamel.yaml-0.18.10/ruamel.yaml.egg-info/requires.txt#line.2